manipulation_object_types:
- {
      "type": "squadra_piccola",
      "grasp_poses": [{
          "position": [0, 0, 0.06],
          "quaternion": [1, 0, 0, 0],
          "approach_distance": [0, 0, 0.1],
          "leave_distance": [0, 0, 0.1],
          "tool": "open_tip",
          "property_pre_exec_id": "open",
          "property_exec_id": "close",
          "property_post_exec_id": "close"
      }, {
          "position": [0, 0, 0.06],
          "quaternion": [0, 1, 0, 0],
          "approach_distance": [0, 0, 0.1],
          "leave_distance": [0, 0, 0.1],
          "tool": "open_tip",
          "property_pre_exec_id": "open",
          "property_exec_id": "close",
          "property_post_exec_id": "close"
      }]
  }
- {
      "type": "squadra_grande",
      "grasp_poses": [{
          "position": [0, 0, 0.06],
          "quaternion": [1, 0, 0, 0],
          "approach_distance": [0, 0, 0.1],
          "leave_distance": [0, 0, 0.1],
          "tool": "open_tip",
          "property_pre_exec_id": "open",
          "property_exec_id": "close",
          "property_post_exec_id": "close"
      }, {
          "position": [0, 0, 0.06],
          "quaternion": [0, 1, 0, 0],
          "approach_distance": [0, 0, 0.1],
          "leave_distance": [0, 0, 0.1],
          "tool": "open_tip",
          "property_pre_exec_id": "open",
          "property_exec_id": "close",
          "property_post_exec_id": "close"
      }]
  }
- {
      "type": "cube",
      "grasp_poses": [{
          "position": [0, 0, 0.01],
          "quaternion": [0, 0, 0, 1],
          "approach_distance": [0, 0, 0.1],
          "leave_distance": [0, 0, 0.1],
          "tool": "open_tip",
          "property_pre_exec_id": "open",
          "property_exec_id": "close",
          "property_post_exec_id": "close"
      }, {
          "position": [0, 0, 0.01],
          "quaternion": [0, 0, 1, 0],
          "approach_distance": [0, 0, 0.1],
          "leave_distance": [0, 0, 0.1],
          "tool": "open_tip",
          "property_pre_exec_id": "open",
          "property_exec_id": "close",
          "property_post_exec_id": "close"
      }, {
          "position": [0, 0, 0.01],
          "quaternion": [0, 0, 0.707107, 0.707107],
          "approach_distance": [0, 0, 0.1],
          "leave_distance": [0, 0, 0.1],
          "tool": "open_tip",
          "property_pre_exec_id": "open",
          "property_exec_id": "close",
          "property_post_exec_id": "close"
      }, {
          "position": [0, 0, 0.01],
          "quaternion": [0, 0, -0.707107, 0.707107],
          "approach_distance": [0, 0, 0.1],
          "leave_distance": [0, 0, 0.1],
          "tool": "open_tip",
          "property_pre_exec_id": "open",
          "property_exec_id": "close",
          "property_post_exec_id": "close"
      }]
  }
- {
      "type": "pin",
      "grasp_poses": [{
          "position": [0, 0, 0.005],
          "quaternion": [1, 0, 0, 0],
          "approach_distance": [0, 0, 0.1],
          "leave_distance": [0, 0, 0.1],
          "tool": "open_tip",
          "property_pre_exec_id": "open",
          "property_exec_id": "close",
          "property_post_exec_id": "close"
      }]
  }
